packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -520,9 +520,7 @@
                                 ></span>
                             </oui-action-menu-item>
 
-                            <oui-action-menu-item
-                                data-ui-sref="app.hosting.dashboard.database.order-private"
-                            >
+                            <oui-action-menu-item href="#/order-cloud-db">
```

Comment:
```suggestion
                            <oui-action-menu-item data-href="#/order-cloud-db">
```

Reply:
thanks for the review. element is modified. I actually remplaced href by data-ui-sref